packages/components/src/components/RunAction/RunAction.js

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2019-2021 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { injectIntl } from 'react-intl';
+import { Button } from 'carbon-components-react';
+import { Play32 as Play, Restart32 as Restart } from '@carbon/icons-react';
+
+class RunAction extends Component {
+  handleAction = event => {
+    event.preventDefault();
+    const {
+      action,
+      getURL,
+      intl,
+      run,
+      runaction,
+      showNotification
+    } = this.props;
+    const { namespace } = run.metadata;
+    let triggerMsg = null;
+    switch (action) {
+      case 'rerun':
+        triggerMsg = intl.formatMessage({
+          id: 'dashboard.rerun.triggered',
+          defaultMessage: 'Triggered rerun'
+        });
+        break;
+      case 'start':
+        triggerMsg = intl.formatMessage({
```

Comment:
I'm not seeing this notification appear when clicking the Start button. Looks like there's an error being thrown from `runaction` and the error handling in turn is failing on a missing `error.response.status` for the error message.

Reply:
It was due to no newRun / no need for the link to new run, as startPipelineRun doesn't return a new run.  I modified to make the link conditional:

https://github.com/tektoncd/dashboard/pull/2106/files#diff-95ec2ca7d82a45cf2f79f910de125bff1d3dfc12d870d5f261967ec47bd92e6aR52-R60